docs/source/reference/core/configuration.rst

```diff
@@ -68,7 +68,15 @@ Note that the default values are set in the global config.
        - If it is ``'auto'``, Chainer will use cuDNN only if it is known that the usage does not degrade the performance.
        - If it is ``'never'``, Chainer will never use cuDNN anywhere.
 
-   The default value is ``'auto'``.
+   The default value is given by ``CHAINER_USE_CUDNN`` environment variable (set to ``'always'``, ``'auto'`` or ``'never'``) if available, otherwise uses ``'auto'``.
+``chainer.config.cudnn_use_tensor_core``
```

Comment:
TensorCore part should not be backported.

Reply:
Thanks. Fixed.